app/controllers/users/invitations_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Users::InvitationsController < Devise::InvitationsController
+  before_action :configure_permitted_parameters
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:invite, keys: [:user_category])
+    devise_parameter_sanitizer.permit(:accept_invitation, keys: [:name])
```

Comment:
Here, we might also need to keys for password and confirm_password

Reply:
already added in permit params by device inevitable